Snakefile

```diff
@@ -0,0 +1,465 @@
+        eyeLocal=f"tmp/EyeG2P-{timeStamp}.csv.gz",
+        skinBucket=GS.remote(f"{config['Gene2Phenotype']['inputBucket']}/SkinG2P-{timeStamp}.csv.gz"),
+        skinLocal=f"tmp/SkinG2P-{timeStamp}.csv.gz",
+        cancerBucket=GS.remote(f"{config['Gene2Phenotype']['inputBucket']}/CancerG2P-{timeStamp}.csv.gz"),
+        cancerLocal=f"tmp/CancerG2P-{timeStamp}.csv.gz"
+    conda:
+        'envs/conda-env.yml'
+    log:
+        GS.remote(logFile)
+    shell:
+        """
+        curl  {params.webSource_dd_panel} > {output.ddLocal}
+        gsutil cp {output.ddLocal} {output.ddBucket}
+        curl  {params.webSource_eye_panel} > {output.eyeLocal}
+        gsutil cp {output.eyeLocal} {output.eyeBucket}
+        curl  {params.webSource_skin_panel} > {output.skinLocal}
+        gsutil cp {output.skinLocal} {output.skinBucket}
+        curl  {params.webSource_cancer_panel} > {output.cancerLocal}
+        gsutil cp {output.cancerLocal} {output.cancerBucket}
+        """
+
+## fetchCrispr           : fetches three tables from GS
+rule fetchCrispr:
+    input:
+        evidenceFile=GS.remote(f"{config['CRISPR']['inputBucket']}/crispr_evidence.tsv"),
+        descriptionsFile=GS.remote(f"{config['CRISPR']['inputBucket']}/crispr_descriptions.tsv"),
+        cellTypesFile=GS.remote(f"{config['CRISPR']['inputBucket']}/crispr_cell_lines.tsv")
+    output:
+        evidenceFile=f"tmp/crispr_evidence-{timeStamp}.csv",
+        descriptionsFile=f"tmp/crispr_descriptions-{timeStamp}.tsv",
+        cellTypesFile=f"tmp/crispr_cell_lines-{timeStamp}.tsv"
+    conda:
+        'envs/conda-env.yml'
+    log:
+        GS.remote(logFile)
+    shell:
+        """
+        gsutil cp {input.evidenceFile} {output.evidenceFile}
+        gsutil cp {input.descriptionsFile} {output.descriptionsFile}
+        gsutil cp {input.cellTypesFile} {output.cellTypesFile}
+        """
+
+## fetchProgeny           : fetches PROGENy table from GS, and two disease-mapping and pathway-mapping look-up tables
+rule fetchProgeny:
+    input:
+        inputFile=GS.remote(f"{config['PROGENy']['inputBucket']}/progeny_normalVStumor_opentargets.txt"),
+        diseaseMapping=config['PROGENy']['diseaseMapping'],
+        pathwayMapping=config['PROGENy']['pathwayMapping']
+    output:
+        inputFile=f"tmp/progeny_normalVStumor_opentargets-{timeStamp}.txt",
+        diseaseMapping=f"tmp/progeny_cancer2EFO_mappings-{timeStamp}.tsv", # solve ambiguity between progeny and slapenrich
+        pathwayMapping=f"tmp/pathway2Reactome_mappings-{timeStamp}.tsv"
+    conda:
+        'envs/conda-env.yml'
+    log:
+        GS.remote(logFile)
+    shell:
+        """
+        cp  {input.diseaseMapping} {output.diseaseMapping}
+        cp  {input.pathwayMapping} {output.pathwayMapping}
+        gsutil cp {input.inputFile} {output.inputFile}
+        """
+
+## fetchSysbio           : fetches evidence data and study level information from GS
+rule fetchSysbio:
```

Comment:
I notice there are pairs of steps, such as `sysbio` / `fetchSysbio`. Can't we unify them for simplicity? Or is there some reason I don't yet understand?

Reply:
@ireneisdoomed What do you think? Should we add that to the follow-up ticket or is this not worth the effort at all?

Reply:
This is an interesting question, when we were brainstorming about the pipeline, we often argued back and forth. Now, I'm not sure we need to do it at all. I'm working with some of the genetics portal pipelines, and there are some interesting implementations: in [this snakefile](https://github.com/opentargets/genetics-v2d-data/blob/master/snakefiles/study_and_top_loci_tables.Snakefile) input files are not explicitly fetched, instead, they just referred and snakemake handles in the background:

```python
from snakemake.remote.FTP import RemoteProvider as FTPRemoteProvider
from snakemake.remote.GS import RemoteProvider as GSRemoteProvider
from snakemake.remote.HTTP import RemoteProvider as HTTPRemoteProvider
```

I think we should do the same. @ireneisdoomed , please take a look at these files, without diving into them too deep, just for getting some inspiration. 

But again: if the current implementation works, it's not a dealbreaker. We'll collect these points under a ticket and we'll schedule a meeting to discuss priorities and what we can drop.

Reply:
Added to the follow-up ticket.